src/create-preview-tab-effects.js

```diff
@@ -44,7 +44,16 @@ function createPreviewTabEffects(PreviewTab, uc) {
 
         autoCrop(store, settings, uc)
 
-        uc.locale.translations = configureTranslations(uc.locale.translations)
+        const currentLocale = settings.locale
+        const {translations, isLocaleExists} = configureTranslations(uc.locale.translations, currentLocale)
+
+        uc.locale.translations = translations
+
+        // Add fallback if locale doesn't exists
+        if (!isLocaleExists) {
+          uc.locale.translations[currentLocale] = translations.en
+        }
```

Comment:
You totally rewrite locale for the preview tab with `en`. You need to use the `en` locale for effects only if custom locale doesn‘t exist in Effects Tab.

Preview tab without Effects Tab on Poland:
<img width="600" alt="2019-02-13 16 48 09" src="https://user-images.githubusercontent.com/948688/52716397-b7be6000-2faf-11e9-8d15-027dba158a7e.png">

Preview tab with Effects Tab on Poland (title and main buttons on english too):
<img width="600" alt="2019-02-13 16 48 45" src="https://user-images.githubusercontent.com/948688/52716430-cc9af380-2faf-11e9-858e-485a89e23d43.png">